lib/mangadex/internal/with_attributes.rb

```diff
@@ -58,22 +60,38 @@ def self.attributes_to_inspect
             Relationship.from_data(relationship_data, MangadexObject.new(**data))
           end
 
-          found_attributes = data['attributes'] || {}
-          attributes = klass.new(**Hash(found_attributes.symbolize_keys))
+          if direct
+            symbolized_data = data.symbolize_keys
+            symbolized_data.transform_keys! do |key|
+              Mangadex::Utils.underscore(key)
+            end
+            keys = symbolized_data.keys
+            keys.each do |key|
+              attr_accessor key
+            end
 
-          initialize_hash = {
-            id: data['id'],
-            type: data['type'] || self.type,
-            attributes: attributes,
-            related_type: related_type,
-          }
+            instance = new
+            keys.each do |key|
+              instance.send("#{key}=", symbolized_data[key])
+            end
 
-          relationships = [source_obj].compact unless relationships.present?
-          initialize_hash.merge!({relationships: relationships}) if relationships.present?
+            instance
+          else
+            found_attributes = data['attributes'] || {}
+            attributes = klass.new(**Hash(found_attributes.symbolize_keys))
+
+            initialize_hash = {
+              id: data['id'],
+              type: data['type'] || self.type,
+              attributes: attributes,
+              related_type: related_type,
+            }
 
-          # binding.pry
```

Comment:
Can remove this comment @thedrummeraki

Reply:
@royalgiant Looks like the line you're referencing was already removed so I think we're good